brave/browser-laptop
# Adding fireball search engine to Brave

## Testing Instructions

1. Check `about:preferences#search`, verify entry for fireball search exists
2. Select fireball as the search engine, and enter a query in the URL bar to verify that search is completed successfully.
